### PR TITLE
[Compute AG] Agentless Secret Detection

### PR DESCRIPTION
**Pull request by rcaballeromx**

### CODE REVIEW

File: compute/admin_guide/compliance/detect_secrets.adoc [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
+== Detect Secrets
 
 Prisma Cloud can detect sensitive information that is improperly secured inside images and containers.
 Scans can detect embedded passwords, login tokens, and other types of secrets.
 To detect improperly secured secrets, add the following checks to your xref:../compliance/manage_compliance.adoc#[compliance policy].
 
 
-[.section]
-=== Compliance check ID 424
+=== Defender Secret Detection
```
Comment (omri86) [inline compute/admin_guide/compliance/detect_secrets.adoc]: Secret -> Secrets

File: compute/admin_guide/compliance/detect_secrets.adoc [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
+
+Your deployed defenders detect sensitive information inside images and containers.
+Select *Monitor > Compliance > Compliance Explorer* and filter the results using the `secrets` keyword to see the impacted images and containers.
```
Comment (omri86) [inline compute/admin_guide/compliance/detect_secrets.adoc]: filtering with "secrets" won't give the the relevant checks, a proper way to look would be to search for the relevant checks you listed below by IDs

File: compute/admin_guide/compliance/detect_secrets.adoc [hunk 3]
```diff
@@ -55,10 +56,87 @@ image::detect_secrets_762507.png[width=800]
...
 
-image::detect_secrets_762508.png[width=800]
+==== Compliance Check ID 597
 
+This check detects sensitive information provided in environment variables of container.
 
-[.section]
-=== Compliance check ID 597
+[#agentless-secret-detection]
+=== Agentless Secret Detection
```
Comment (omri86) [inline compute/admin_guide/compliance/detect_secrets.adoc]: Secret -> Secrets

File: compute/admin_guide/compliance/detect_secrets.adoc [hunk 4]
```diff
@@ -55,10 +56,87 @@ image::detect_secrets_762507.png[width=800]
...
 
-This check detects sensitive information provided in environment variables of container.
+Agentless scanning detects sensitive information inside your hosts and container images.
+
+==== Compliance Check ID 456
+
+This check detects secrets inside your hosts' filesystem.
+By default, Prisma Cloud alerts you that a secret was detected in a host.
+Select *Monitor > Compliance > Compliance Explorer* and filter the results using `secret` as a keyword to see the impacted hosts.
```
Comment (omri86) [inline compute/admin_guide/compliance/detect_secrets.adoc]: filter using the ID

File: compute/admin_guide/compliance/detect_secrets.adoc [hunk 5]
```diff
@@ -55,10 +56,87 @@ image::detect_secrets_762507.png[width=800]
...
+
+==== Compliance Check ID 457
+
+This check detects secrets inside your container images' filesystem.
+By default, Prisma Cloud alerts you that a secret was detected in a container image.
+Select *Monitor > Compliance > Compliance Explorer* and filter the results using `secret` as a keyword to see the impacted container images.
```
Comment (omri86) [inline compute/admin_guide/compliance/detect_secrets.adoc]: filter using the ID

File: compute/admin_guide/compliance/detect_secrets.adoc [hunk 6]
```diff
@@ -55,10 +56,87 @@ image::detect_secrets_762507.png[width=800]
...
+** AWS MWS Auth Token
+
+* Azure
+
+** Azure Storage Account access key
+** Azure Service Principal
+
+* GCP Service Account Auth Key
+
+* The private keys of encryption keys
```
Comment (omri86) [inline compute/admin_guide/compliance/detect_secrets.adoc]: Private encryption keys

File: compute/admin_guide/compliance/detect_secrets.adoc [hunk 7]
```diff
@@ -55,10 +56,87 @@ image::detect_secrets_762507.png[width=800]
...
+This check detects secrets inside your container images' filesystem.
+By default, Prisma Cloud alerts you that a secret was detected in a container image.
+Select *Monitor > Compliance > Compliance Explorer* and filter the results using `secret` as a keyword to see the impacted container images.
+
+[#detected-secrets]
+==== Detected Secrets
+
+When agentless scanning detects a secret in a host or container image, only the following information is sent to the Prisma Cloud console.
+
+* A redacted snippet of the secret with a length of less than 10 characters.
```
Comment (omri86) [inline compute/admin_guide/compliance/detect_secrets.adoc]: * A redacted snippet of the secret with a length of less than 10 characters.
    * 10% visible characters for secrets above 30 characters up to a maximum of 10 characters.
    * Maximum of 3 visible characters for secrets under 30 characters.

File: compute/admin_guide/compliance/detect_secrets.adoc [hunk 8]
```diff
@@ -55,10 +56,87 @@ image::detect_secrets_762507.png[width=800]
...
+* The filename where the secret was detected.
```
Comment (omri86) [inline compute/admin_guide/compliance/detect_secrets.adoc]: The path to the file where the secret was detected.

File: compute/admin_guide/compliance/detect_secrets.adoc [hunk 9]
```diff
@@ -55,10 +56,87 @@ image::detect_secrets_762507.png[width=800]
...
+* The line where the secret was detected.
```
Comment (omri86) [inline compute/admin_guide/compliance/detect_secrets.adoc]: The line and offset in the file where the secret was detected in the form of `line:offset`.

File: compute/admin_guide/compliance/detect_secrets.adoc [hunk 10]
```diff
@@ -55,10 +56,87 @@ image::detect_secrets_762507.png[width=800]
...
+* The offset in the line where the secret was detected.
```
Comment (omri86) [inline compute/admin_guide/compliance/detect_secrets.adoc]: remove considering my previous comment

File: compute/admin_guide/compliance/detect_secrets.adoc [hunk 11]
```diff
@@ -55,10 +56,87 @@ image::detect_secrets_762507.png[width=800]
...
+==== Compliance Check ID 457
+
+This check detects secrets inside your container images' filesystem.
+By default, Prisma Cloud alerts you that a secret was detected in a container image.
+Select *Monitor > Compliance > Compliance Explorer* and filter the results using `secret` as a keyword to see the impacted container images.
+
+[#detected-secrets]
+==== Detected Secrets
+
+When agentless scanning detects a secret in a host or container image, only the following information is sent to the Prisma Cloud console.
```
Comment (omri86) [inline compute/admin_guide/compliance/detect_secrets.adoc:83]: this is missing:
* The secret type (see list below) <- maybe link this to the relevant section that lists the types
* The modified time of the file where the secret was detected.

File: compute/admin_guide/compliance/detect_secrets.adoc [hunk 12]
```diff
@@ -55,10 +56,87 @@ image::detect_secrets_762507.png[width=800]
...
+
+* A redacted snippet of the secret with a length of less than 10 characters.
+* The filename where the secret was detected.
+* The line where the secret was detected.
+* The offset in the line where the secret was detected.
+
+This information provides you with enough context to find the detected secret and take any needed action.
+
+Agentless scanning detects the following secrets.
```
Comment (omri86) [inline compute/admin_guide/compliance/detect_secrets.adoc]: following secrets -> following secrets types

File: compute/admin_guide/compliance/detect_secrets.adoc [hunk 13]
```diff
@@ -55,10 +56,87 @@ image::detect_secrets_762507.png[width=800]
...
 
+This check detects sensitive information provided in environment variables of container.
 
-[.section]
-=== Compliance check ID 597
+[#agentless-secret-detection]
+=== Agentless Secret Detection
 
-This check detects sensitive information provided in environment variables of container.
+Agentless scanning detects sensitive information inside your hosts and container images.
```
Comment (omri86) [inline compute/admin_guide/compliance/detect_secrets.adoc]: Agentless scanning detects sensitive information inside files in your hosts and container images filesystem.

**Review by omri86 — CHANGES_REQUESTED** [review]
LGTM good job @rcaballeromx thanks for the quick turnaround!

**Comment by rcaballeromx** [comment]
@omri86 I addressed all your feedback. Thank you for the quick review.
@Pubs-MV I've tested the content and it's ready to be merged.

**Pull request merged**